src/App.tsx

```diff
@@ -1,14 +1,27 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodoId, setTodoId] = useState<number | null>(null);
+  const [filter, setFilter] = useState<string>(Filter.all);
```

Comment:
leave default type (Filter)
```suggestion
  const [filter, setFilter] = useState(Filter.all);
```

Reply:
if i do that i get an error